.travis.yml

```diff
@@ -1,15 +1,19 @@
 # whitelist
 branches:
   only:
+      - master
       - tickets/DM-5218
 
-sudo: required
+sudo: false 
```

Comment:
I believe `required` is technically required in order to use the docker service, although the travis gem linter does not warn on this.